ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1135,6 +1135,7 @@ IE_CPU.fused_clamp_float
 
 # Result mismatch 2.666666507720947266 is not close to 3 at index 1
```

Comment:
@ilyachur Can you look at it? It is probably a plugin bug.

Reply:
Discussed offline. In general, first of all we should detect the component which has a bug. It can be ONNX importer, Conversion passes or plugin.

Reply:
CVS-35806